### PR TITLE
DD-8 home page kafelki

### PR DESCRIPTION
**Pull request by AgataHalasDroptica**

### CODE REVIEW

File: themes/custom/droopler_theme/scss/layout/_modifiers.scss [hunk 1]
```diff
@@ -225,3 +227,55 @@ $stripe-background-color: rgba(0,0,0, 0.57) !default;
...
+    }
+    &.background-image {
+      z-index: -1 !important;
+    }
+  }
+
+  .paragraph {
+    h2 {
+      text-align: center;
+      color: $white;
```
Comment (grzegorz-pietrzak-droptica) [inline themes/custom/droopler_theme/scss/layout/_modifiers.scss]: Let's use `$color-odysseus` here (hope I remember the name, please check in `_colors.scss` :))

File: themes/custom/droopler_theme/scss/layout/_modifiers.scss [hunk 2]
```diff
@@ -225,3 +227,55 @@ $stripe-background-color: rgba(0,0,0, 0.57) !default;
...
+      font-size: 4vh;
+      @include media-breakpoint-down(md) {
+        font-size: 3vh;
+      }
+      &:after {
+        background: $white;
```
Comment (grzegorz-pietrzak-droptica) [inline themes/custom/droopler_theme/scss/layout/_modifiers.scss]: Here too

File: themes/custom/droopler_theme/scss/layout/_modifiers.scss [hunk 3]
```diff
@@ -225,3 +227,55 @@ $stripe-background-color: rgba(0,0,0, 0.57) !default;
   }
 }
 
+.tile {
+  padding: $image-background-padding-y 0 $tile-title-padding-y 0;
+  position: sticky;
```
Comment (grzegorz-pietrzak-droptica) [inline themes/custom/droopler_theme/scss/layout/_modifiers.scss]: Does the tile stay at the top of the screen when scrolled down?

File: themes/custom/droopler_theme/scss/layout/_modifiers.scss [hunk 4]
```diff
@@ -225,3 +227,55 @@ $stripe-background-color: rgba(0,0,0, 0.57) !default;
...
+    &.background-image {
+      z-index: -1 !important;
+    }
+  }
+
+  .paragraph {
+    h2 {
+      text-align: center;
+      color: $white;
+      font-size: 4vh;
```
Comment (grzegorz-pietrzak-droptica) [inline themes/custom/droopler_theme/scss/layout/_modifiers.scss]: Let's add those two values as vars

File: themes/custom/droopler_theme/scss/layout/_modifiers.scss [hunk 5]
```diff
@@ -15,6 +15,9 @@ $arrow-margin-top: 6% !default;
 $arrow-border: 9px solid transparent !default;
 $arrow-down-color: lighten($color-helen, 10%) !default;
 $stripe-background-color: rgba(0,0,0, 0.57) !default;
+$image-background-padding-y: 110px !default;
```
Comment (grzegorz-pietrzak-droptica) [inline themes/custom/droopler_theme/scss/layout/_modifiers.scss]: Maybe we should use a name like `tile-padding-top`? But I may not understand the context :)

File: themes/custom/droopler_theme/scss/layout/_modifiers.scss [hunk 6]
```diff
@@ -225,3 +227,55 @@ $stripe-background-color: rgba(0,0,0, 0.57) !default;
...
+    h2 {
+      text-align: center;
+      color: $white;
+      font-size: 4vh;
+      @include media-breakpoint-down(md) {
+        font-size: 3vh;
+      }
+      &:after {
+        background: $white;
+        margin: 7px auto;
```
Comment (grzegorz-pietrzak-droptica) [inline themes/custom/droopler_theme/scss/layout/_modifiers.scss]: Why do we need a margin on after? (just a question :))

Comment (AgataHalasDroptica) [reply themes/custom/droopler_theme/scss/layout/_modifiers.scss]: It is for underline under the title

Comment (grzegorz-pietrzak-droptica) [reply themes/custom/droopler_theme/scss/layout/_modifiers.scss]: ok, clear

**Pull request merged**